scss/_variables.scss

```diff
@@ -788,8 +788,12 @@ $mark-bg-dark:                $white !default; // Boosted mod
 // scss-docs-start table-variables
 $table-cell-padding-y:      .875rem !default;
 $table-cell-padding-x:      $spacer * .5 !default;
+$table-cell-icon-margin-top:        -.688rem !default; // Boosted mod
+$table-cell-icon-margin-bottom:     -.438rem !default; // Boosted mod
 $table-cell-padding-y-sm:   .5625rem !default;
 $table-cell-padding-x-sm:   $table-cell-padding-x !default;
+$table-cell-icon-margin-top-sm:     -.563rem !default; // Boosted mod
+$table-cell-icon-margin-bottom-sm:  -.313rem !default; // Boosted mod
```

Comment:
With the actual nomenclature I'd called those variables `$table-sm-cell-icon-margin-bottom` but the others are made this way 🤔